src/sentry/tasks/post_process.py

```diff
@@ -161,6 +161,16 @@ def post_process_group(event, is_new, is_regression, is_sample, is_new_group_env
                             event=event,
                         )
 
+        if event.get_event_type() == 'error' and features.has(
```

Comment:
ideally we'd only execute this if we thought there was going to be work done (e.g. they had an integration setup that would need it)

Reply:
hmm so that gets done here https://github.com/getsentry/sentry/blob/master/src/sentry/tasks/sentry_apps.py#L151 - not sure how much time that adds to `post_process`